app/decorators/manageiq/providers/openshift/container_manager_decorator.rb

```diff
@@ -1,7 +1,11 @@
 module ManageIQ::Providers::Openshift
   class ContainerManagerDecorator < MiqDecorator
-    def listicon_image
+    def self.listicon_image
       "svg/vendor-openshift.svg"
     end
+
+    def listicon_image
+      this.class.listicon_image
```

Comment:
Probably not. `self` might work instead of `this`, but please do check that it works :).